GarhalController/GarhalController.vcxproj

```diff
@@ -109,6 +109,7 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
```

Comment:
No need to standarize a C++ version

Reply:
I only applied the settings from "Release | x64" to other configurations. So that switching the configuration or platform does not break the assembly.

Reply:
Valid.